Lombiq.HelpfulLibraries.Cli/Lombiq.HelpfulLibraries.Cli.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Helpful Libraries - Command Line Libraries</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2011, Lombiq Technologies Ltd.</Copyright>
```

Comment:
Let's use our time machine and travel back to the future:
```suggestion
    <Copyright>Copyright © 2022, Lombiq Technologies Ltd.</Copyright>
```

Reply:
OHNO 😱 https://www.youtube.com/watch?v=S4KKf84I6M0

Every HL project has `Copyright © 2011` so I don't think we should. @Piedone could you confirm?

Reply:
I would amend [the wiki](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/edit-v2/19496967) to say this (emphasis is the proposed addition). Is that accurate?

> The Copyright year should be the same as in the License file, adjust if necessary. **For sub-projects it should be the same as the main project, the year of the initial commit.**